src/transporters.ts

```diff
@@ -60,7 +58,7 @@ export class DefaultTransporter {
    */
   configure(opts: GaxiosOptions = {}): GaxiosOptions {
     opts.headers = opts.headers || {};
-    if (!isBrowser()) {
+    if (typeof window === 'undefined') {
```

Comment:
we're okay to keep this check the same?

Reply:
Aye.  In this case, it doesn't have an impact on which browser feature we're trying to use.  It's just a naive check so we can try to send the right HTTP headers (I think).